node-src/index.ts

```diff
@@ -302,7 +306,7 @@ export async function getGitInfo(): Promise<GitInfo> {
     uncommittedHash,
     userEmail,
     userEmailHash,
-    repositoryRootDir,
+    repositoryRootDir: repositoryRootDirectory,
```

Comment:
Is this return something we can change, or is it in public scope?

Reply:
I think we'd be able to change it but it'll be a pain since the output is used in [chromaui/addon-visual-tests](https://github.com/chromaui/addon-visual-tests). Perhaps I should leave a comment about that here. I had to go hunting to figure out why this exists but isn't called anywhere.